test/plugin/test_out_detect_exceptions.rb

```diff
@@ -46,11 +46,17 @@ def create_driver(conf = CONFIG, tag = DEFAULT_TAG)
     d
   end
 
-  def feed_lines(driver, t, *messages)
+  def log_entry(message, count, stream)
+    log_entry = { 'message' => message, 'count' => count }
+    log_entry['stream'] = stream unless stream.nil?
+    log_entry
+  end
+
+  def feed_lines(driver, t, messages, stream: nil)
```

Comment:
Since `stream` as a named argument (rather than a positional with a default value), you should be able to keep `*messages` (they shouldn't conflict) and save yourself a whole lot of square brackets.